src/annotator/guest.js

```diff
@@ -367,6 +352,17 @@ export default class Guest {
       this.setHighlightsVisible(showHighlights);
     });
 
+    this._sidebarRPC.on(
+      'deleteAnnotation',
+      /** @type {string} */ tag => this.detach(tag)
```

Comment:
Using `@type` here doesn't work, and unfortunately TS doesn't complain if a JSDoc annotation is not successfully attached to a value. This works:

```js
    this._sidebarRPC.on(
      'deleteAnnotation',
      /** @param {string} tag */
      tag => this.detach(tag)
    );
```

Reply:
Yes, sorry. I have made similar mistakes it works if it is wrapped in parenthesis.
`(/** @type {string} */ tag) => this.detach(tag)`, but I have forgotten to add the parenthesis.

I